plugin/input/k8s/multiline_action.go

```diff
@@ -71,11 +72,17 @@ func (p *MultilineAction) Do(event *pipeline.Event) pipeline.ActionResult {
 		// check buffer size before append
 		if p.maxEventSize == 0 || sizeAfterAppend < p.maxEventSize {
 			p.eventBuf = append(p.eventBuf, logFragment[1:logFragmentLen-1]...)
-		} else if !p.skipNextEvent {
-			// skip event if max_event_size is exceeded
-			p.skipNextEvent = true
-			ns, pod, _, _, _ := getMeta(event.SourceName)
-			p.logger.Errorf("event chunk will be discarded due to max_event_size, source_name=%s, namespace=%s, pod=%s", event.SourceName, ns, pod)
+		} else {
+			if p.controller != nil {
```

Comment:
nit: I guess it is useful , example https://github.com/ozontech/file.d/blob/master/plugin/action/join/join.go#L145

Reply:
nil-check for some tests